sway-lib-std/src/tx.sw

```diff
@@ -218,17 +241,25 @@ pub fn tx_output_pointer(index: u64) -> u32 {
 }
 
 /// Get the type of an output given a pointer to the output.
-pub fn tx_output_type(ptr: u32) -> u8 {
+pub fn tx_output_type_from_pointer(ptr: u32) -> u8 {
     asm(r1, r2: ptr) {
         lw r1 r2 i0;
         r1: u8
     }
 }
 
+/// Get the type of an output at a given index
+pub fn tx_output_type(index: u64) -> u8 {
+    let ptr = tx_output_pointer(index);
+    let output_type = tx_output_type_from_pointer(ptr);
+    output_type
```

Comment:
```suggestion
    tx_output_type_from_pointer(ptr)
```

Reply:
Resolved in [a7c7a91](https://github.com/FuelLabs/sway/pull/2142/commits/a7c7a91776162bd5d4a0bbbbf94c14c066c83e9a)